### PR TITLE
ToDo: Add item 

### PR DESCRIPTION
**Pull request by samiullahbahadur**
# Add, Remove and update items

- I'm done all of things the below:
- Implement a function for adding a new task (add a new element to the array).
- Implement a function for deleting a task (remove an element from the array).
- Deleting a task should update all remaining items' indexes.
- A function for editing task descriptions is implemented.
- All changes to the To-Do List should be saved in local storage

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+          <div class="input-todo">
+            <input
+              class="add-todo"
+              type="text"
+              placeholder="Add to your list..."
+              required
+            />
+            <button type="submit" class="submit">
+            <img
+              src="../src/asset/return-down-back-outline.svg"
```
Comment (er-contreras) [inline src/index.html]: - [ ] It'll be great if we can see the image working properly to give a good look to our project. 😉

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,116 @@
+import './style.css';
+
+const list = document.querySelector('.list');
+const inputTodo = document.querySelector('.add-todo');
+const addTodoBtn = document.querySelector('.submit');
+
+class TodoTask {
+  constructor(desp, arr, done = false) {
```
Comment (er-contreras) [inline src/index.js]: - [ ] It'd be great if we can add arguments that other people that are looking at your code for the first time can understand what is about. Please change 'desp' argument for another more readable.

**Review by er-contreras — CHANGES_REQUESTED** [review]
Hi @samiullahbahadur, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

almost there

## Required Changes ♻️
- [ ] A function for editing task descriptions is implemented
  - All changes to the To-Do List should be saved in local storage.
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.html (hunk 1 above)
Comment (samiullahbahadur) [reply src/index.html]: yes, you right .
I'm did fix it

File: src/index.js (hunk 2 above)
Comment (samiullahbahadur) [reply src/index.js]: yes, you right .
I'm did change it

**Review by rustam-tolipov — APPROVED** [review]
## Hi @samiullahbahadur 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉, 

Cheers and Happy coding!👏👏👏

Developer-512px-10

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**